### PR TITLE
update azure storage account client log

### PR DESCRIPTION
**Pull request by lovebaby979**
<!--  Thanks for sending a pull request!  Here are some tips for you: 
update azure storage account client log

#### Which issue(s) this PR fixes:
Fixes #
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
```docs
update azure storage account client log
```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 1]
```diff
@@ -358,14 +358,14 @@ func (c *Client) ListStorageAccountByResourceGroup(ctx context.Context, resource
 	resp, rerr := c.armClient.GetResource(ctx, resourceID, "")
 	defer c.armClient.CloseResponse(ctx, resp)
 	if rerr != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.list.request", resourceID, rerr.Error())
+		klog.V(5).InfoS("Received error in storageAccount.list.request", "resourceID", resourceID,"error",  rerr.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:361]:
```suggestion
		klog.V(5).InfoS("Received error in storageAccount.list.request", "resourceID", resourceID, "error",  rerr.Error())
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 2]
```diff
@@ -135,7 +135,7 @@ func (c *Client) getStorageAccount(ctx context.Context, resourceGroupName string
 		azure.WithErrorUnlessStatusCode(http.StatusOK),
 		autorest.ByUnmarshallingJSON(&result))
 	if err != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageaccount.get.respond", resourceID, err)
+		klog.V(5).Infof("Received error in storageaccount.get.respond", "resourceID", resourceID, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:138]:
```suggestion
		klog.V(5).InfoS("Received error in storageaccount.get.respond", "resourceID", resourceID, "error", err)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 3]
```diff
@@ -196,7 +196,7 @@ func (c *Client) listStorageAccountKeys(ctx context.Context, resourceGroupName s
 		azure.WithErrorUnlessStatusCode(http.StatusOK),
 		autorest.ByUnmarshallingJSON(&result))
 	if err != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageaccount.listkeys.respond", resourceID, err)
+		klog.V(5).InfoS("Received error in storageaccount.listkeys.respond", "resourceID", resourceID, "error", err)
```
Comment (andyzhangx) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:199]: this is changing the format of original logging, what's the benefit?

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 4]
```diff
@@ -126,7 +126,7 @@ func (c *Client) getStorageAccount(ctx context.Context, resourceGroupName string
 	response, rerr := c.armClient.GetResource(ctx, resourceID, "")
 	defer c.armClient.CloseResponse(ctx, response)
 	if rerr != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageaccount.get.request", resourceID, rerr.Error())
+		klog.V(5).InfoS("Received error in storageaccount.get.request", "resourceID", resourceID, "error", rerr.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:129]:
```suggestion
		klog.V(5).InfoS("Received error in storageaccount.get.request", "resourceID", resourceID, "err", rerr.Error())
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go (hunk 3 above)
Comment (yangjunmyfm192085) [reply staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:199]:
```suggestion
		klog.V(5).InfoS("Received error in storageaccount.listkeys.respond", "resourceID", resourceID, "err", err)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 5]
```diff
@@ -247,14 +247,14 @@ func (c *Client) createStorageAccount(ctx context.Context, resourceGroupName str
 	response, rerr := c.armClient.PutResource(ctx, resourceID, parameters)
 	defer c.armClient.CloseResponse(ctx, response)
 	if rerr != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.put.request", resourceID, rerr.Error())
+		klog.V(5).InfoS("Received error in storageAccount.put.request", "resourceID", resourceID, "error", rerr.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:250]:
```suggestion
		klog.V(5).InfoS("Received error in storageAccount.put.request", "resourceID", resourceID, "err", rerr.Error())
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 6]
```diff
@@ -247,14 +247,14 @@ func (c *Client) createStorageAccount(ctx context.Context, resourceGroupName str
...
 		return rerr
 	}
 
 	if response != nil && response.StatusCode != http.StatusNoContent {
 		_, rerr = c.createResponder(response)
 		if rerr != nil {
-			klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.put.respond", resourceID, rerr.Error())
+			klog.V(5).InfoS("Received error in storageAccount.put.respond", "resourceID", resourceID, "error", rerr.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:257]:
```suggestion
			klog.V(5).InfoS("Received error in storageAccount.put.respond", "resourceID", resourceID, "err", rerr.Error())
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 7]
```diff
@@ -358,14 +358,14 @@ func (c *Client) ListStorageAccountByResourceGroup(ctx context.Context, resource
...
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.list.request", resourceID, rerr.Error())
+		klog.V(5).InfoS("Received error in storageAccount.list.request", "resourceID", resourceID,"error",  rerr.Error())
 		return result, rerr
 	}
 
 	var err error
 	page.alr, err = c.listResponder(resp)
 	if err != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.list.respond", resourceID, err)
+		klog.V(5).InfoS("Received error in storageAccount.list.respond", "resourceID", resourceID, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:368]:
```suggestion
		klog.V(5).InfoS("Received error in storageAccount.list.respond", "resourceID", resourceID, "err", err)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go [hunk 8]
```diff
@@ -378,7 +378,7 @@ func (c *Client) ListStorageAccountByResourceGroup(ctx context.Context, resource
 		}
 
 		if err = page.NextWithContext(ctx); err != nil {
-			klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageAccount.list.next", resourceID, err)
+			klog.V(5).InfoS("Received error in storageAccount.list.next", "resourceID", resourceID, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go:381]:
```suggestion
			klog.V(5).InfoS("Received error in storageAccount.list.next", "resourceID", resourceID, "err", err)
```

**Pull request closed without merging**